types/types.ts

```diff
@@ -98,69 +137,69 @@ export type Config = {
 };
 
 export interface MainnetContracts {
-  [core: string]: ethers.Contract;
-  tribe: ethers.Contract;
-  fei: ethers.Contract;
-  uniswapPCVDeposit: ethers.Contract;
+  core: Core;
```

Comment:
why remove [core: string]? I remember needing that for something

Reply:
Doesn't make any sense to me to have an array of cores' in this, and it doesn't seem to be breaking anything.